egs/europarl_st/SRT/lcma_srt/train.py

```
@@ -0,0 +1,2196 @@
+        with torch.no_grad():
+            model_avg.load_state_dict(
+                (model.module if hasattr(model, "module") else model).state_dict()
+            )
+    saved_params: Dict[str, Any] = {}
+    reset_progress = getattr(params, "reset_progress_stats", False)
+    if params.resume_optimizer_scheduler_scaler and not reset_progress:
+        if "optimizer" in rest and optimizer is not None:
+            logging.info("Loading optimizer state dict")
+            try:
+                optimizer.load_state_dict(rest["optimizer"])
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)